.travis.yml

```diff
@@ -6,6 +6,6 @@ os:
 language: node_js
 node_js:
   - 'node'
-  - '11'
-  - '10'
-  - '8'
+  - '14'
+  - '13'
+  - '12'
```

Comment:
Changed to pass the tests because the ESM syntax does not work on NodeJS < 12. 
It should be possible to test the bundled version with a lower version of NodeJS, tell me if I should do it.